### ISSUE TITLE
Celery Workers occasionally exit prematurely for no obvious reason

### ISSUE THREAD

**Issue by timothydereuse** [DDMAL/Rodan#670]
Errors like this happen inconsistently - not just on text alignment in GPU-celery but when starting other jobs, in other containers. Have not figured out a way to consistently reproduce this. Whenever this happens the container must be restarted, which is not difficult and happens automatically, but it means that some job in Rodan hangs forever.

```
[2022-06-10 10:08:01,828: INFO/MainProcess] Received task: Text Alignment[50a09898-c8d4-49d1-89c5-4d6397af7c41]  
[2022-06-10 10:08:01,901: INFO/ForkPoolWorker-1] Text Alignment[50a09898-c8d4-49d1-89c5-4d6397af7c41]: {'OCR Model': 0}
[2022-06-10 10:08:05,936: INFO/ForkPoolWorker-1] Text Alignment[50a09898-c8d4-49d1-89c5-4d6397af7c41]: processing image...
[2022-06-10 10:08:05,936: WARNING/ForkPoolWorker-1] identifying text lines...
[2022-06-10 10:09:21,326: ERROR/MainProcess] Process 'ForkPoolWorker-1' pid:109 exited with 'signal 9 (SIGKILL)'
[2022-06-10 10:09:21,400: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 9 (SIGKILL) Job: 2.')
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/billiard/pool.py", line 1267, in mark_as_worker_lost
    human_status(exitcode), job._job),
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 9 (SIGKILL) Job: 2.
```

We need to figure out how to reproduce this behavior. Please leave a message in this issue if you notice this happening somewhere else. Perhaps there is a pattern to what kills the celery container.

**Comment by timothydereuse** [comment]
We increased the amount of memory allocated by docker to `rodan-main` from 2G to 4G. If things stop crashing I'll close this issue.

**Comment by softcat477** [comment]
Fix in PRs #816 #803 #799 #798 #792. Closing this issue.
